dozer/cogs/profile_menus.py

```diff
@@ -10,42 +16,92 @@ class ProfileMenus(commands.Cog):
     """
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
-        self.ctx_menu = app_commands.ContextMenu(
-            name = 'View Profile',
-            callback = self.profile,  # sets the callback the view_profile function
-        )
-        self.bot.tree.add_command(self.ctx_menu)  # add the context menu to the tree
-
-    async def cog_unload(self) -> None:
```

Comment:
Why was this deleted?

Reply:
This one only allowed one context menu per class/cog, updated version allows for multiple commands. It also makes more sense snice the format is similar to regular commands